transit_odp/timetables/tests/test_csv.py

```diff
@@ -401,6 +501,10 @@ def test_stale_12_months_old(effective, modified, period_end, is_stale):
         df["Timeliness Status"][0] == "Service hasn't been updated within a year"
     ) == is_stale
     assert df["Requires Attention"][0] == "Yes" if is_stale else "No"
+    assert (
+        df["Date when data is over 1 year old"][0]
+        == (txc.modification_datetime + timedelta(days=365)).date()
+    )
```

Comment:
Not in this line but line 584 there is a failure when running automated test cases

Reply:
Fixed that test case